spark/src/test/scala/org/apache/spark/sql/delta/skipping/clustering/IncrementalZCubeClusteringSuite.scala

```diff
@@ -281,5 +283,159 @@ class IncrementalZCubeClusteringSuite extends QueryTest
       }
     }
   }
+
+  test("OPTIMIZE FULL") {
```

Comment:
can we add a test case for different clusteringProvider with OPTIMIZE FULL?

Reply:
Added a new test `OPTIMIZE FULL - change clustering provider`